test/functional/Java8andUp/playlist.xml

```diff
@@ -2562,5 +2562,30 @@
 			<impl>ibm</impl>
 		</impls>
 	</test>
+	
+	<test>
+		<testCaseName>LoadClassWithUTF8PkgNameTest</testCaseName>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) $(JAVA_COMMAND) $(JVM_OPTIONS) \
+			-cp $(Q)$(LIB_DIR)$(D)asm-all.jar$(P)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(Q) \
+			org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) \
+			-testnames LoadClassWithUTF8PkgNameTest \
+			-groups $(TEST_GROUP) \
+			-excludegroups $(DEFAULT_EXCLUDE); \
+			$(TEST_STATUS)
+		</command>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<subsets>
+			<subset>8+</subset>
+		</subsets>
```

Comment:
If it is 8+, we do not need this subsets at all. By default, it is 8+

Reply:
Removed the subset as suggested above.